### DISCUSSION TITLE
How to add multiple h265 camera streams and convert to h264?

### DISCUSSION THREAD

**Discussion by djronh1** [bluenviron/mediamtx#1974] [Questions]
Hi,

I have several security cameras that only support h265, and I'm trying to convert their rtsp streams to h264
with the goal of using them with Scyrpted and then set them up for HKSV (Apple HomeKit Secure Video).

I'm trying to setup first two cameras.

System that I'm running mediamtx supports both nVidia encode/decode as well as Intel QuickSync 

Based on search, I've added this:

paths:
#all:
drivewayh265:
  source: rtsp://<user>:<password>T@192.168.0.41:554/Streaming/Channels/101
  runOnReady: ffmpeg -i rtsp://localhost:$RTSP_PORT/drivewayh265 -pix_fmt yuv420p -c:v h264_nvenc -preset fast -b:v 600k -max_muxing_queue_size 1024 -f rtsp rtsp://localhost:$RTSP_PORT/drivewayh264
shedh265:
  source: rtsp://<user>:<password>T@192.168.0.42:554/Streaming/Channels/101
  runOnReady: ffmpeg -i rtsp://localhost:$RTSP_PORT/shedh265 -pix_fmt yuv420p -c:v h264_nvenc -preset fast -b:v 600k -max_muxing_queue_size 1024 -f rtsp rtsp://localhost:$RTSP_PORT/shedh264
  runOnReadyRestart: yes

But i'm unable to connect from remote system via ffplay or vlc.

ffplay replay returns rtsp://192.168.0.51:8554/shedh264: Server returned 400 Bad Request
vlc log has
live555 error: Failed to connect with rtsp://192.168.0.51:8554/shedh264
satip error: Failed to setup RTSP session

Any suggestions on how to setup?

**Comment by aler9** [comment]
Hello, the `runOnReady` lines looks good to me, but the log you posted refers to handshake issues between FFmpeg / live555 and the server. Check the server log in order to find out what the problem is. If you still had not figured out the issue, post here the full server log.

[a comment by an automated account is not shown]
